src/building/prerequisites.md

```diff
@@ -0,0 +1,29 @@
+# Prerequisites
+
+Before building the compiler, you need the following things installed:
+
+* Python
+* A C/C++ compiler toolchain
+* cmake
+* rustc
```

Comment:
I'm intrigued by this... what do we need rustc for? compiling the `rustbuild` tool? The actual compiler and `std` builds are done by a downloaded artifact.